reference/library/src/morpheus-master/js/src/sakai.morpheus.more.sites.js

```diff
@@ -368,8 +368,8 @@ $PBJQ(document).ready(function($){
   };
 
   var renderFavoriteCount = function () {
-    // Subtract 1 from the count to avoid counting "My Workspace", which can't be moved anyway.
-    var favoriteCount = $PBJQ('.site-favorite', favoritesPane).length - 1;
+    // SAK-31246: it is not needed to subtract 1 from the count to avoid counting "My Workspace" due to SAK-31134.
```

Comment:
Generally it's nearly always better to explain this in the commit comment as otherwise over time the code becomes littered with SAK references.